Kernel/Output/PDF/Ticket.pm

```diff
@@ -184,6 +186,17 @@ sub GeneratePDF {
         );
     }
 
+    if ( $Interface{Customer} ) {
+        %PrintingCustomerData = $CustomerUserObject->CustomerUserDataGet(
+            User => $Param{UserID},
+        );
+    }
+
+    if ( !%PrintingCustomerData )
```

Comment:
Looks better. Did you run the CodePolicy on that code? I think that Perl::Critic would put the opening brace at the end of line 195.

Reply:
CodePolicy didn't change anything - made this change by myself.